src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import LoginOrSignup from '../pages/LoginOrSignup';
+
+class App extends React.Component {
+    render() {
+        return (
+            <React.Fragment>
+               <CssBaseline/>
+                 <LoginOrSignup/>
+            </React.Fragment>
```

Comment:
@KalariaYash use two tab space indent or can set it in the code for all .js file